internal/handlers/common/msg_listcommands.go

```diff
@@ -107,6 +107,10 @@ var Commands = map[string]command{
 		Help:    "Returns a summary of all runtime and configuration options.",
 		Handler: (handlers.Interface).MsgGetCmdLineOpts,
 	},
+	"getFreeMonitoringStatus": {
+		Help:    "MsgGetFreeMonitoringStatus returns the status of free Cloud monitoring.",
```

Comment:
Doesn't it has to be in a small letter? 
```suggestion
		Help:    "MsgGetFreeMonitoringStatus returns the status of free cloud monitoring.",
```